docs/source/error_code_list2.rst

```diff
@@ -231,31 +231,29 @@ since unless implemented by a sub-type, the expression will always evaluate to t
     if foo:
        ...
 
+This is similar in concept to ensuring that an expression's type implements an expected interface (e.g. ``Sized``),
+except that attempting to invoke an undefined method (e.g. ``__len__``) results in an error,
+while attempting to evaluate an object in boolean context without a concrete implementation results in a truthy value.
```

Comment:
I don't understand what this sentence is trying to say. What is "this" exactly?

Reply:
As best as I can tell, "this" refers to the check itself and the fact that it doesn't highlight a concrete issue but rather only a potential one.